python/src/aiconfig/editor/client/src/utils/aiconfigStateUtils.ts

```diff
@@ -47,3 +47,21 @@ export function getModelSettingsStream(
 
   return undefined;
 }
+
+export function convertClientAIConfigToAIConfig(
```

Comment:
We already have this as `clientConfigToAIConfig` in the types (just have it colocated with the definitions of ClientAIConfig, etc.)